package.json

```diff
@@ -70,29 +71,29 @@
     "karma-mocha-reporter": "^2.2.2",
     "karma-remap-istanbul": "^0.6.0",
     "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^2.0.2",
+    "karma-webpack": "^3.0.0",
     "npm-run-all": "^4.0.2",
     "raw-loader": "^1.0.0-beta.0",
     "rimraf": "^2.6.1",
     "rollup": "^0.58.2",
     "rollup-globals-regex": "^0.0.3",
     "rollup-plugin-node-resolve": "^3.0.0",
-    "rxjs": "^5.5.0",
+    "rxjs": "^6.1.0",
     "semantic-release": "^15.1.4",
     "source-map": "^0.7.2",
     "source-map-loader": "^0.2.0",
     "sourcemap-istanbul-instrumenter-loader": "^0.2.0",
-    "tslib": "^1.8.0",
-    "tslint": "^5.8.0",
-    "tslint-loader": "^3.4.3",
-    "typescript": "~2.4.0",
-    "uglify-js": "^3.0.15",
-    "webpack": "^3.8.1",
-    "webpack-dev-server": "^2.9.3",
-    "zone.js": "^0.8.18"
+    "tslib": "^1.9.0",
+    "tslint": "^5.10.0",
+    "tslint-loader": "^3.6.0",
+    "typescript": "~2.7.2",
+    "uglify-js": "^3.3.24",
+    "webpack": "^4.8.1",
+    "webpack-dev-server": "^3.1.4",
+    "zone.js": "^0.8.16"
   },
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=8.0.0"
```

Comment:
I'm not sure if bumping node engine is required.

As we might break some people who are using node v6

Reply:
Angular CLI 6 requires that

Reply:
What is people don't use angular cli?

Reply:
I thought this project is using Angular CLI, I upgraded few libraries and get lost a bit, I'll revert that change soon.